admin/settings.php

```diff
@@ -126,23 +126,30 @@ public function init_settings() {
 				if(!isset($field["register"]) || $field["register"]) {
 					register_setting($this->option_group, $this->prefix.$field["name"]);
 				}
-				
+
 			}
 		}
 	}
-	
+
 	/**
 	 * Add Settings submenu and page
 	 *
 	 * @since  0.10
 	 * @return void
 	 */
 	public function add_settings_menu() {
+
+		$cap = RUA_App::CAPABILITY;
+		if ( ! is_multisite() && is_super_admin() && ! current_user_can( $cap ) ) {
+			// Replicate multisite super admin behavior for single installations.
+			$cap = 'manage_options';
```

Comment:
I see what you mean, and it looks better now.

I thought about making this a filter to `user_has_cap`, so that whenever we check for `RUA_App::CAPABILITY`, it will also check if the user is super admin, e.g. kind of like this:
```

function user_is_level_admin( $allcaps, $cap, $args ) {
	if($cap == RUA_App::CAPABILITY && is_super_admin()) {
		$allcaps[RUA_App::CAPABILITY] = 1;
	}
	return $allcaps;
}
```

If you look at `has_cap`:
https://developer.wordpress.org/reference/classes/wp_user/has_cap/

You can see that on multisites, the function is already returned before reaching the user_has_cap filter, so there shouldn't be a need to check for that.

What do you think of this approach? Obviously the filter has to be applied after the already existing `user_level_has_cap.`

So instead of calling our own `is_level_admin()`, we can continue using `current_user_can` and rely on the manage_restrictions capability.